ubuntu22.04/precompiled/nvidia-driver

```diff
@@ -210,10 +211,19 @@ _unload_driver() {
 
 # Link and install the kernel modules from a precompiled packages
 _install_driver() {
-    echo "Installing NVIDIA driver kernel modules..."
-    apt-get install --no-install-recommends -y linux-objects-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION} \
-    linux-signatures-nvidia-${KERNEL_VERSION} \
-	linux-modules-nvidia-${DRIVER_BRANCH}-server-${KERNEL_VERSION}
+    if [ "$USE_OPEN_KERNEL_MODULES" = true ]; then
```

Comment:
Let's change the variable name to OPEN_KERNEL_MODULES_ENABLED to align with our non-precompiled images: https://github.com/NVIDIA/gpu-driver-container/blob/main/ubuntu22.04/nvidia-driver#L19